utils/a11y_controller.js

```diff
@@ -780,7 +781,7 @@ export default class A11yController {
     }
 
     handleFocus = (event) => {
-        if (!this.mouseIsPressed && this.windowIsFocused) {
+        if (this.lastInputEventIsKeyboard && this.windowIsFocused) {
```

Comment:
This is the fix for the multiple post hover menus. Prevents focus handling if not using the keyboard.